tests/test_vllm_client_server.py

```diff
@@ -99,6 +99,26 @@ def test_generate(self):
         for seq in completion_ids:
             assert all(isinstance(tok, int) for tok in seq)
 
+    def test_chat(self):
```

Comment:
@qgallouedec I'm sorry but I'm not able to run this test.
Could you please give me some hint about the environment requirements so I can run it?
Thanks! 🤗

Reply:
We might have to mock the response?